### DISCUSSION TITLE
Improve password generation

### DISCUSSION THREAD

**Discussion by itamaroryan** [FIRSTIsrael/lems#724] [Ideas]
Right now, passwords are generated completely randomly by a utility function in [utils-random.ts](https://github.com/FIRSTIsrael/lems/blob/main/libs/utils/random/src/lib/utils-random.ts). This often results in cumbersome passwords, such as those including `I,l`, `0,O` and such. We should find a better way to generate random strings for these passwords.

**Comment by johnmeshulam** [comment]
Alternatively, generating the passwords as images or with a legible font can work! For example, in Consolas each letter is clearly visually distinct. `I, l` vs. I, l and `0, O` vs 0, O
